<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>505</width>
    <height>505</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>505</width>
    <height>505</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>461</width>
         <height>141</height>
        </rect>
       </property>
       <property name="title">
        <string>Plot appearance</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>31</y>
          <width>201</width>
          <height>105</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Plot style:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="plotStyleSelectorBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Line style:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="lineStyleSelectorBox">
             <item>
              <property name="text">
               <string>solid</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>dashed</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>dashdot</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>dotted</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Line width:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="lineWidthSelectorBox">
             <item>
              <property name="text">
               <string>0.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1.0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2.0</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="enableMarkersBox">
           <property name="text">
            <string>Enable markers</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>30</y>
          <width>197</width>
          <height>105</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Cursor style:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cursorStyleSelectorBox">
             <item>
              <property name="text">
               <string>solid</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>dashed</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>dashdot</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>dotted</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Cursor width:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cursorWidthSelectorBox">
             <item>
              <property name="text">
               <string>0.25</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>0.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>0.75</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1.0</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Cursor color:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cursorColorSelectorBox">
             <item>
              <property name="text">
               <string>red</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>green</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>blue</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>cyan</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>magenta</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>yellow</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>black</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="enableMplToolbarBox">
           <property name="text">
            <string>Use generic toolbar (deprecated)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modeSelectorBox">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Simple CSV reader</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>J1939 dump decoder</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="simple_csv">
      <attribute name="title">
       <string>Simple CSV reader</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>461</width>
         <height>371</height>
        </rect>
       </property>
       <property name="title">
        <string>Apply scale to</string>
       </property>
       <widget class="QTableWidget" name="scaleTable">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>441</width>
          <height>301</height>
         </rect>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Scale</string>
         </property>
        </column>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>330</y>
          <width>351</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="addScaleButton">
           <property name="text">
            <string>Add value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeScaleButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearScaleButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>11</y>
         <width>96</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Alias for delimiter:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="simpleCsvAliasEdit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>11</y>
         <width>100</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="simpleCsvTimestampEdit">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>10</y>
         <width>100</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>104</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Timestamp column:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="dump_decoder">
      <attribute name="title">
       <string>J1939 dump decoder</string>
      </attribute>
      <widget class="QGroupBox" name="CanDumpAscBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>311</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>ASC format settings</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Base of ID/data:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="ascBaseSelector">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>hex</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dec</string>
         </property>
        </item>
       </widget>
       <widget class="QCheckBox" name="relTimestampBox">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Relative timestamp</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>461</width>
         <height>341</height>
        </rect>
       </property>
       <property name="title">
        <string>Database setup</string>
       </property>
       <widget class="QListWidget" name="databaseList">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>441</width>
          <height>271</height>
         </rect>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>300</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="addDatabaseButton">
           <property name="text">
            <string>Add database files...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearDatabaseButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
